### PR TITLE
Move object command 

### PR DESCRIPTION
**Pull request by t-jenh**

### CODE REVIEW

File: src/code.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import {handleScroll} from './utilities/scrollLib';
 import { Z_NO_COMPRESSION } from 'zlib';
 import { copiedSelections, copySelection, pasteSelection, duplicateSelection} from './utilities/copyPasteDuplicate';
 import {handleBringSendObject} from './utilities/bringSendObject';
+import { selectSelection } from './utilities/selectionLib';
```
Comment (tiffanyjc) [inline src/code.ts:12]: No need to import this -- it's already imported in line 5 as selectionLib! 
So you can call `selectionLib.selectSelection`

File: src/code.ts [hunk 2]
```diff
@@ -72,27 +73,24 @@ figma.ui.onmessage = async (msg) => {
...
       yCorr = object.y - msg.magnitude
-    } else if (msg.Direction === "down") {
+    } else if (msg.direction.toString() === 'down') {
       yCorr = object.y + msg.magnitude
     }
     object.relativeTransform = [[1,0,xCorr],[0,1,yCorr]]
-
-    //example below
-    //var rect2 = figma.createRectangle()
-    // rect2.relativeTransform = [[1,0,30],[0,1,40]]
```
Comment (tiffanyjc) [inline src/code.ts:95]: Any chance we can grab all this code (it looks good!) and put it into a separate .ts files under the **utilities** folder? Take a look at `zoomLib.ts`and the `msg.type === 'zoom' ` to see what I mean. It'll make the code cleaner / fewer merge conflicts later :)

Comment (t-jenh) [reply src/code.ts]: Done!

File: src/code.ts (hunk 1 above)
Comment (t-jenh) [reply src/code.ts]: Deleted

**Pull request merged**